clients/python/tests/test_api.py

```diff
@@ -56,19 +59,146 @@ def test_post_model_version_artifacts(auth_headers: dict, artifact_type: str, ur
     payload = {
         "artifactType": artifact_type,
         "name": "my-test-model-artifact-post",
-        "uri": f"{uri_prefix}my-test-model.pkl",
         "state": state,
         "description": "A test model artifact created via direct POST test.",
         "externalId": str(secrets.randbelow(2000000000 - 100000 + 1) + 100000)
     }
 
+    # Add type-specific properties
+    if artifact_type == "model-artifact":
+        payload["modelFormatName"] = "tensorflow"
+        payload["modelFormatVersion"] = "1.0"
+        payload["uri"] = f"{uri_prefix}my-test-model.pkl"
+    elif artifact_type == "doc-artifact":
+        payload["uri"] = f"{uri_prefix}documentation.pdf"
+    elif artifact_type == "dataset-artifact":
+        payload["uri"] = f"{uri_prefix}dataset.parquet"
+        payload["sourceType"] = "s3"
+        payload["source"] = "s3://test-bucket/datasets/"
+    elif artifact_type == "metric":
+        payload["value"] = 0.95  # Example accuracy value
+        # Use current time in milliseconds plus random offset to ensure uniqueness
+        import time
+        current_ms = int(time.time() * 1000)
+        random_offset = secrets.randbelow(1000000)  # Random offset up to 1 million ms (1000 seconds)
+        payload["timestamp"] = str(current_ms + random_offset)
+    elif artifact_type == "parameter":
+        payload["value"] = "0.001"  # Example learning rate
+        payload["parameterType"] = "string"
+
```

Comment:
maybe a support function can be used here

Reply:
Added helper functions